README.md

```diff
@@ -1,8 +1,72 @@
-This is an example project to demonstrate the usage of Neon branching Github actions 
+# neon_twitter
 
-## Getting Started
+`neon_twitter` is a microblogging application designed to demonstrate the database branching capability of Neon Serverless Postgres with Neon's [GitHub Actions](https://neon.tech/docs/guides/branching-github-actions#create-branch-action).
 
-First, run the development server:
+Neon provides the following GitHub Actions for working with Neon branches, which can be integrated into your CI workflows:
+
+- [Create branch action](https://github.com/neondatabase/create-branch-action)
+- [Delete branch action](https://github.com/neondatabase/delete-branch-action)
+
+This application leverages GitHub Actions to automate branch creation, schema validation with Prisma, and migration deployments. Each pull request to the main branch triggers the creation of a temporary branch to test Prisma migrations, ensuring schema changes are valid. When changes are pushed to the main branch, these migrations are applied to the database schema.
+
+## Get Started
+
+### 1. Clone the repository
+
+```bash
+git clone git@github.com:neondatabase/neon_twitter.git
+cd neon_twitter
+```
+
+### 2. Start with a fresh history
+
+Remove the original git history:
+
+```bash
+rm -rf .git
+```
+
+### 3. Initialize your new repository
+
+```bash
+git init
+```
+
+### 4. Push to your own GitHub repository
+
+First, create a new repository on GitHub. Once created:
+
+```bash
+git remote add origin git@github.com:[your-username]/your_neon_twitter.git
+git add .
+git commit -m "Initial commit"
+git push -u origin master
+```
+
+Replace [your-username] with your GitHub username.
+
+### 5. Set up your shadow database
+
+Create a database named shadow in Neon. For instructions, see [Create a database](https://neon.tech/docs/manage/databases#create-a-database).
+
+### 6. Configure your local environment
+
+Create a `.env` file in the root of the project. Replace the placeholders ([user], [password], [neon_hostname], [dbname]) with your actual values:
+
+```env
+DATABASE_URL="postgres://[user]:[password]@[neon_hostname]/[dbname]"
+DIRECT_URL="postgres://[user]:[password]@[neon_hostname]/[dbname]"
+SHADOW_DATABASE_URL="postgres://[user]:[password]@[neon_hostname]/shadow"
+```
+
+### 7. Install dependencies and migrate
+
+```bash
+npm install
```

Comment:
I've used a yarn for this repo, so there is a yarn.lock file. May be more sense to use it for examples:
```
yarn 
yarn run prisma migrate dev
```

Reply:
Fixed! Thank you.